src/sentry/static/sentry/app/components/panels/panelItemGroup.jsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'react-emotion';
+
+import PanelItem from 'app/components/panels/panelItem';
+
+const PanelItemGroup = styled.div`
```

Comment:
Let's do `styled('div')` to be consistent when styling another component.

Reply:
Is this what we're trying to do commonly now?